Configuration/AlCa/python/autoCond.py

```diff
@@ -20,7 +20,7 @@
     # GlobalTag for Run1 data reprocessing
     'run1_data'         :   'GR_R_74_V12A::All',
     # GlobalTag for Run2 data reprocessing
-    'run2_data'         :   'GR_R_74_V13A::All',
+    'run2_data'         :   '74X_dataRun2_Prompt_v1::All',
```

Comment:
@hengne you should change 'Configuration/AlCa/python/autoCond_condDBv2.py'. autoCond.py still maps to v1 Global Tags (and actually '74X_dataRun2_Prompt_v1::All' doesn't even exists in the condition database V1. @diguida